README.md

```diff
@@ -74,6 +74,7 @@ Common configuration that you may want to change includes:
 To generate a tarball archive that includes the OpenTelemetry binary and a configuration file compatible with Google Cloud Monitoring:
 
 1. Run `make build-tarball`.
+  - To also include JVM, Apache, MySQL and StasD support, run `make build-tarball-exporters` instead
```

Comment:
Note four spaces represents an indent whereas two does not, so this will render better as (also fixes typo):
```suggestion
    - To also include JVM, Apache, MySQL and StatsD support, run `make build-tarball-exporters` instead
```